extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -330,7 +358,7 @@ public static boolean areHardcodedYoutubeMusicKeysValid() throws IOException, Re
                 .end()
                 .value("query", "test")
                 .value("params", "Eg-KAQwIARAAGAAgACgAMABqChAEEAUQAxAKEAk%3D")
-            .end().done().getBytes("UTF-8");
+            .end().done().getBytes(StandardCharsets.UTF_8);
```

Comment:
Please don't do this, it will break NewPipe Legacy unfortunately

Reply:
wow, the cleanup feature on commit from Android Studio changed that to StandardCharsets when i changed it to the String.